odoo/tools/mail.py

```diff
@@ -595,3 +600,21 @@ def formataddr(pair, charset='utf-8'):
                 name=email_addr_escapes_re.sub(r'\\\g<0>', name),
                 addr=address)
     return address
+
+def localpartify(email_name):
+    """
+    Turn any ascii string into a rfc2822 compliant local-part.
+
+    >>> 'john.doe, MyCompany Inc.'
+    'john.doe-MyCompany-Inc'
+    """
+    email_name.encode('ascii')  # would require SMTP-UTF8 otherwise
```

Comment:
Is that for the assertion that the email_name is ascii only?

Reply:
Really ? Thought it was clear enough :(

Reply:
It's really not clear that it's an *assertion*, so at first glance it looks like a leftover no-op. `encode` is not usually invoked for its side-effects.

Though pycharm doesn't seem to flag it (unlike accessing properties / fields for the  side-effects).